include/spm/rel/an2_08.h

```diff
@@ -44,7 +44,8 @@ typedef struct An2_08Work {
 } An2_08Work; //sizeof 0x98
 SIZE_ASSERT(An2_08Work, 0x98);
 
-extern An2_08Work An2_08_wp;
+extern An2_08Work an2_08_wp;
+extern const char * wang_special_1_80def2c8[4];
```

Comment:
For 'unnamed' things, stick to lbl_{addr} rather than ghidra's names for them (simplifies things in decomp since they're automatically named that there)